fpga/src/bootrom/src/main.c

```diff
@@ -25,6 +25,11 @@ int main() {
         "la a1, device_tree;");
       //"ebreak;");
 
+    if(read_csr(mhartid)>1) {
+      while(1)
+        __asm__ volatile ("wfi;");
```

Comment:
We should add support for quad-core in the runtime (@ricted98) and in the bootrom (@maicolciani) so that it can be tested bare metal

Reply:
Should we keep it for a separate PR? @maicolciani

Reply:
yes we will make a separe PR.